src/vuln_analysis/utils/intel_retriever.py

```diff
@@ -85,6 +88,19 @@ async def _get_session(self, session: aiohttp.ClientSession | None = None):
             async with aiohttp.ClientSession() as session:
                 yield session
 
+    def _get_timeout_description(self) -> str:
+        """
+        Get a human-readable description of the effective timeout.
+
+        Returns:
+            A string describing the timeout configuration
+        """
+        if self._intel_source_timeout is not None:
+            return f"{self._intel_source_timeout} seconds"
```

Comment:
I really like that the new log format specifies which setting(s) triggered the timeout. Thoughts on adding it here too?
```suggestion
            return f"{self._intel_source_timeout} seconds (intel_source_timeout={self._intel_source_timeout})"
```